src/components/DataItemExplorerFolderStructure/index.tsx

```diff
@@ -55,18 +55,22 @@ const FolderStructure = (): JSX.Element => {
     if (!selectedId) {
       return;
     }
-    getRequestHandler(
-      url,
-      `?id=${selectedId}`,
-      ({ children }: DataItemExplorerItem) => {
-        children &&
-          setTreeStructure((treeStructure) => getTreeStructure(treeStructure, children, treeLevel));
-      },
-      (error: Error): void => {
-        // eslint-disable-next-line no-console
-        console.log(error);
-      }
-    );
+    async (): Promise<void> => {
+      await getRequestHandler(
+        url,
+        `?id=${selectedId}`,
+        ({ children }: DataItemExplorerItem) => {
+          children &&
+            setTreeStructure((treeStructure) =>
+              getTreeStructure(treeStructure, children, treeLevel)
+            );
+        },
+        (error: Error): void => {
+          // eslint-disable-next-line no-console
+          console.log(error);
```

Comment:
Thoughts on adding a logger?

Reply:
Yeh, this was just temporary for now. Think a logger would be good